src/validation.h

```diff
@@ -85,10 +85,9 @@ static const int MAX_SCRIPTCHECK_THREADS = 15;
 static const int DEFAULT_SCRIPTCHECK_THREADS = 0;
 /** Number of headers sent in one getheaders result. We rely on the assumption that if a peer sends
  *  less than this number, we reached its tip. Changing this value is a protocol upgrade. */
+static const unsigned int MAX_HEADERS_RESULTS = 8000;
```

Comment:
Should not use old name for better error resistance for backports. So, it won't be mistakenly used directly this constant instead helper function